api/api.go

```diff
@@ -217,10 +233,15 @@ func getMeshStatus(api *API) meshStatus {
 	return strippedStatus
 }
 
-func (api *API) alertGroups(w http.ResponseWriter, req *http.Request) {
-	var err error
-	matchers := []*labels.Matcher{}
-	if filter := req.FormValue("filter"); filter != "" {
+func (api *API) alertGroups(w http.ResponseWriter, r *http.Request) {
```

Comment:
@stuartnelson3 Are the changes on the `alertGroups` function and all subsequent changes still needed?